tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py

```diff
@@ -15,7 +15,7 @@
 import subprocess
 import sys
 
-sys.path.append("software-management-end-to-end")
+sys.path.append("software_management_end_to_end")
```

Comment:
You could avoid this in this file and other similar tests by just adding `<pythonpath value="${testRootDir}/software_management_end_to_end" />` to `pysysproject.xml` file just like it's done for other environments.

Reply:
Good hint I did that for the tests that use the line above.